lib/clients/web/facets/users.js

```diff
@@ -75,6 +76,21 @@ UsersFacet.prototype.list = function list(opts, optCb) {
   return this.makeAPICall('users.list', null, opts, optCb);
 };
 
+/**
+ * Find a user with an email address.
+ * @see {@link https://api.slack.com/methods/users.lookupByEmail|users.lookupByEmail}
+ *
+ * @param {?} email - An email address belonging to a user in the workspace
+ * @param {function=} optCb Optional callback, if not using promises.
+ */
+UsersFacet.prototype.lookupByEmail = function lookupByEmail(email, optCb) {
```

Comment:
can you add an `opts` argument, and pass it through to `makeAPICall()`?

we've been facing a number of issues that stem from Web API methods that gain optional arguments after they are first released, and then we have to break the API to update that method to include another argument.